### PR TITLE
GCI-212 Remove unused log private field from PersonName

### PR DESCRIPTION
**Pull request by JyothsnaAshok**
## Description of what I changed

Unused private fields should be removed .If a private field is declared but not used in the program, it can be considered dead code and should therefore be removed.

## Issue I worked on

see https://issues.openmrs.org/browse/GCI-212

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/PersonName.java [hunk 1]
```diff
@@ -40,7 +38,6 @@
  * A Person can have zero to n PersonName(s).
  */
 @Indexed
-public class PersonName extends BaseChangeableOpenmrsData implements java.io.Serializable, Cloneable, Comparable<PersonName> {
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/PersonName.java]: Did you just remove the class declaration by mistake?
Are you sure this class compiles?

**Comment by JyothsnaAshok** [comment]
@samuelmale Can you please check if it's alright or not?

File: api/src/main/java/org/openmrs/PersonName.java [hunk 2]
```diff
@@ -40,12 +38,11 @@
  * A Person can have zero to n PersonName(s).
  */
 @Indexed
+
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/PersonName.java]: You don't need this whitespace. Could we remove this?

File: api/src/main/java/org/openmrs/PersonName.java [hunk 3]
```diff
@@ -28,8 +28,6 @@
 import org.hibernate.search.annotations.IndexedEmbedded;
 import org.openmrs.api.APIException;
 import org.openmrs.api.db.hibernate.search.LuceneAnalyzers;
-import org.openmrs.layout.name.NameSupport;
-import org.openmrs.layout.name.NameTemplate;
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/PersonName.java]: Did you intend to remove the above two imports?

Comment (JyothsnaAshok) [reply api/src/main/java/org/openmrs/PersonName.java]: That was mentioned in the task description. Should I revert back the deletions?

File: api/src/main/java/org/openmrs/PersonName.java [hunk 4]
```diff
@@ -44,8 +42,6 @@
 	
 	public static final long serialVersionUID = 4353L;
 
-	private static final Logger log = LoggerFactory.getLogger(PersonName.class);
```
Comment (samuelmale) [inline api/src/main/java/org/openmrs/PersonName.java]: Removing this isn't required as a matter of fact that its used here 
https://github.com/openmrs/openmrs-core/blob/c4b7108dfd2aaf39e42fc02d73c07893bf3c08b0/api/src/main/java/org/openmrs/PersonName.java#L407

That's why this file isn't compiling. 

cc: @dkayiwa

Comment (JyothsnaAshok) [reply api/src/main/java/org/openmrs/PersonName.java]: @samuelmale So what should I do now?

**Comment by samuelmale** [comment]
Closing this PR because the issue is no longer valid.

**Comment by dkayiwa** [comment]
Cancelling this because a number of changes have happened since when this was last written. Hence making it non relevant.
Sorry about this @JyothsnaAshok and @samuelmale

**Comment by samuelmale** [comment]
Lets close this PR since primarily the issue itself looks invalid @JyothsnaAshok

**Pull request closed without merging**